asn1ate/sema.py

```diff
@@ -535,8 +547,6 @@ def __str__(self):
 
 class ReferencedValue(SemaNode):
     def __init__(self, elements):
-        # TODO: Module references are not resolved at the moment,
```

Comment:
But now we do know how to handle them, don't we? Any chance you could add an implementation per `DefinedType` here?